### ISSUE TITLE
Paper: Exascale literature study

### ISSUE THREAD

**Issue by romulogoncalves** [NLeSC/TEAM2018#43]
@stijnh Jason has suggested that you will be working on a paper during the Sprint. Could you give us a bit more information about it, little title (issue subject) and what will it be about?

**Comment by stijnh** [comment]
I will continue working on my literature study into exascale computing as part of project PROCESS (EU H2020 project involving Jason).

**Comment by romulogoncalves** [comment]
@stijnh  is there project hours for this work? If yes, which one?

**Comment by stijnh** [comment]
"PROCESS H2020 - PROviding Comp solutions ExaScale challengeS" (33017006) in Exact.

**Comment by romulogoncalves** [comment]
@stijnh were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by stijnh** [comment]
Yes, all goals were achieved. The literature study is part of larger paper and will also be part of my PhD project. I shall close the issue.
